src/Symfony/Component/Console/Question/Question.php

```diff
@@ -274,4 +274,18 @@ protected function isAssoc($array)
     {
         return (bool) \count(array_filter(array_keys($array), 'is_string'));
     }
+
+    public function setQuestion(string $question): self
```

Comment:
`: self` forbids overriding the method from a child class: https://3v4l.org/7Y2sa
It should really be `: Question` with a `@return $this` docblock

Reply:
done